ledger/participant-integration-api/src/test/suite/scala/platform/indexer/parallel/ParallelIndexerSubscriptionSpec.scala

```diff
@@ -162,6 +171,86 @@ class ParallelIndexerSubscriptionSpec extends AnyFlatSpec with Matchers {
     actual shouldBe expected
   }
 
+  behavior of "inputMapper transaction metering"
+
+  {
```

Comment:
since there is only one test case would you mind moving the constants inside of the test and removing the extra curly from here?

Reply:
Done.